detection-rules/attachment_html_js_rc4.yml

```
@@ -0,0 +1,22 @@
+name: "Attachment: RC4 encryption algorithm usage within HTML"
```

for consistency w/ other rule names, this might be:

```suggestion
name: "Attachment: HTML smuggling with RC4 decryption"
```